source/Libraries/SteamLibrary/SteamShared/MetadataProvider.cs

```diff
@@ -445,16 +445,18 @@ internal string ParseDescription(string description)
                     if (ass["type"].Value == "franchise")
                     {
                         var value = ass["name"].Value;
-                        if (publishers.HasItems() && publishers.Any(x => x.Contains(value, StringComparison.OrdinalIgnoreCase)))
+                        if (publishers.HasItems() && (publishers.Any(x => x.Contains(value, StringComparison.OrdinalIgnoreCase) ||

```

Comment:
`publishers.HasItems()` is not necessary if you are going to call `Any` anyways, just add null check `.?Any`.

Reply:
To use a null check, a cast is needed. I personally don't use them but should I change it and use a cast?

![image](https://user-images.githubusercontent.com/1389286/167379328-b7c63196-1a6c-4a69-995b-893de38bd3eb.png)

Reply:
You need to add check specifically for `true`: `publishers?.Any(x => x.Contains(value, StringComparison.OrdinalIgnoreCase) == true`